### PR TITLE
Link to https instead of http

### PR DESCRIPTION
**Pull request by fabianegli**

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -8,13 +8,13 @@ Project dependencies
 
 Pyteomics uses the following Python packages:
 
- - `numpy <http://pypi.python.org/pypi/numpy>`_
- - `matplotlib <http://sourceforge.net/projects/matplotlib/files/matplotlib/>`_
+ - `numpy <https://pypi.python.org/pypi/numpy>`_
+ - `matplotlib <https://sourceforge.net/projects/matplotlib/files/matplotlib/>`_
```
Comment (levitsky) [inline INSTALL:12]:
```suggestion
 - `matplotlib <https://matplotlib.org/>`_
```

Comment (levitsky) [inline INSTALL:12]:
```suggestion
 - `matplotlib <https://matplotlib.org/>`_
```

File: INSTALL [hunk 2]
```diff
@@ -8,13 +8,13 @@ Project dependencies
...
    (used by :py:mod:`pyteomics.pylab_aux`)
- - `lxml <http://pypi.python.org/pypi/lxml>`_ (used by XML parsing modules)
- - `pandas <http://pandas.pydata.org/>`_ (can be used with :py:mod:`pyteomics.pepxml`,
+ - `lxml <https://pypi.python.org/pypi/lxml>`_ (used by XML parsing modules)
```
Comment (levitsky) [inline INSTALL:14]:
```suggestion
 - `lxml <https://lxml.de/>`_ (used by XML parsing modules)
```

**Review by levitsky — CHANGES_REQUESTED** [review]
Thanks! I suggest we also link to the projects' current websites.

File: INSTALL (hunk 1 above)
Comment (fabianegli) [reply INSTALL]: I think this was a duplicate.

File: INSTALL [hunk 3]
```diff
@@ -8,13 +8,13 @@ Project dependencies
 
 Pyteomics uses the following Python packages:
 
- - `numpy <http://pypi.python.org/pypi/numpy>`_
- - `matplotlib <http://sourceforge.net/projects/matplotlib/files/matplotlib/>`_
+ - `numpy <https://pypi.python.org/pypi/numpy>`_
```
Comment (levitsky) [inline INSTALL:11]:
```suggestion
 - `numpy <https://numpy.org/>`_
```

**Review by levitsky — APPROVED** [review]

**Pull request merged**